msteams-platform/m365-apps/teamsjs-support-m365.md

```diff
@@ -0,0 +1,2069 @@
+---
+title: Teams JavaScript client library support across Microsoft 365
+author: erikadoyle
+ms.author: edoyle
+description: Understand the level of support for different TeamsJS library capabilities running in different hosts for Teams apps, including Microsoft Teams, Outlook, and Microsoft 365 app 
+ms.localizationpriority: high
+ms.topic: conceptual
+keywords: TeamsJS Teams JavaScript library capability Microsoft 365 M365
+---
+# TeamsJS capability support across Microsoft 365
+
+Starting with version 2.0.0, Teams JavaScript client library (TeamsJS) enables [certain types of Teams apps](./overview.md) to run across the Microsoft 365 ecosystem. Currently, Microsoft 365 application hosts (Microsoft 365 app and Outlook) for Teams apps support a subset of the application types and capabilities you can build for the Teams platform. This support will expand over time.
+
+This article details the level of support of Teams JavaScript client (TeamsJS) library version 2.x capabilities for *personal scope* tab apps and dialogs (task modules) across various Microsoft 365 host applications. For a more info on what's changed between TeamsJS versions 1.x and 2.x, see [What's new in TeamsJS version 2.x.x](../tabs/how-to/using-teams-client-library.md#whats-new-in-teamsjs-version-2xx).
+
+> [!TIP]
+> You can check for host support of a given capability at runtime by calling the `isSupported()` function on that capability (namespace or subnamespace).
+
+The *TeamsJS Capability* table below lists TeamsJS capabilities (public namespaces) and their support across Microsoft 365 hosts. Some capabilties are marked with  "Deprecated" or "Preview" badges, which have the following meanings:
+
+| Badge | Meaning |
+| - | - |
+| Preview :::image type="content" source="./images/preview-badge.png" alt-text="Image with the word 'Preview' inside a blue rectangle"::: | This capability is in preview and subject to change based on feedback. Please do not use in production. |
+| Deprecated :::image type="content" source="./images/deprecated-badge.png" alt-text="Image with the word 'Deprecated' inside an orange rectangle"::: | This capability is deprecated in favor of newer functionality, though  supported for backwards compability purposes. For new apps, please use the capability recommended in the usage notes of the deprecated capability. |
+
+For readability, Microsoft 365 hosts are signified by the following product icons:
+
+| Microsoft Teams | Microsoft 365 app | Microsoft Outlook |
```

Comment:
Need to add a section for Microsoft Teams built in Angular and the *new* Microsoft Teams built in React. Also, need a new section for the *new* Outlook client which is in the Excel document(Monarch).

Reply:
For simplicity, how about we keep the high-level Teams/Harmony/Outlook icons in the top-level summary table, then go into details like that in the tables of individual capability sections?